ons-spark/ancillary-topics/pandas-udfs.ipynb

```diff
@@ -295,7 +295,9 @@
     "\n",
     "Be aware that scalar UDFs may cause incorrect results (if your dataframe is over multiple partitions) when calculating means and standard deviations. For more info on this and how to overcome this issue see [this towards data science post](https://towardsdatascience.com/pyspark-or-pandas-why-not-both-95523946ec7c).\n",
     "\n",
-    "In the decorator of a scalar Pandas UDF the first arguement is the data type of the output dataframe and the second argument is the UDF type. Here is a simple Pandas scalar UDF to find out the `distance` between the London Fire Bridage (lfb) department and the location of an animal-related incident. Eastings and northings are used as inputs and the output is in 'km'. "
+    "In the decorator of a scalar Pandas UDF the first argument is the data type of the output dataframe and the second argument is the UDF type. Here is a simple Pandas scalar UDF to find out the `distance` between the London Fire Brigade (lfb) and the location of an animal-related incident. The inputs to the function are Eastings and Northings which are types of coordinates that tell you how far East or North a point is; therefore, there are two points inputted for the lfb location and two points inputted for the animal-related incident. The function works to find the distance between these two points and converts the answer into 'km'. \n",
+    "\n",
+    "Please note that the following example has been used here to illustarte how to use a Pandas UDF, this is not necessarily the most efficient way to write this function as it can also be written in PySpark. "
```

Comment:
Sorry just realised I hadn't actually reviewed the changes (caught up in the Git excitement), it looks good - only minor change is just "illustrate" not "illustarte"